getting-started/assets/eclipselink/persistence.xml

```diff
@@ -32,8 +32,7 @@
     <class>org.apache.polaris.jpa.models.ModelSequenceId</class>
     <shared-cache-mode>NONE</shared-cache-mode>
     <properties>
-      <property name="jakarta.persistence.jdbc.url"
-        value="jdbc:postgresql://postgres:5432/{realm}"/>
+      <property name="jakarta.persistence.jdbc.url" value="jdbc:postgresql://postgres:5432/{realm}"/>
```

Comment:
nit: spurious change

Reply:
Actually this is on purpose 😅 it makes the code to replace the JDBC URL as part of the deployment scripts a lot easier to read/understand